src/person/controllers/update_person_controller.py

```diff
@@ -60,6 +65,16 @@ def process(self):
             if possible_error is not None:
                 return possible_error
 
+        if not is_paused:
```

Comment:
regarding my comment above: if `is_paused` is not set in the request body, it will evaluate to `None`, and python will treat it as `False` and execute this if-statement (which we don't want to happen)

instead, do: `if is_paused==False:`